src/rpdk/core/contract/suite/handler_delete.py

```diff
@@ -63,7 +68,10 @@ def contract_delete_list(resource_client, deleted_resource):
     #       remove the model from the list, however.
 
     deleted_model, _request = deleted_resource
-    assert not test_model_in_list(resource_client, deleted_model)
+    assert not test_model_in_list(
+        resource_client, deleted_model
+    ), "any subsequent list operation MUST NOT return the primaryIdentifier\
+         associated with the deleted resource instance"
```

Comment:
Maybe rephrase to something more similar to above like: "A list operation MUST NOT return the primaryIdentifier of any deleted resource instance"